app/Event/EventCommandHandlerProvider.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Event;
+
+use CultuurNet\UDB3\Container\AbstractServiceProvider;
+use CultuurNet\UDB3\Event\CommandHandlers\CopyEventHandler;
+use CultuurNet\UDB3\Event\CommandHandlers\DeleteOnlineUrlHandler;
+use CultuurNet\UDB3\Event\CommandHandlers\RemoveThemeHandler;
+use CultuurNet\UDB3\Event\CommandHandlers\UpdateAttendanceModeHandler;
+use CultuurNet\UDB3\Event\CommandHandlers\UpdateAudienceHandler;
+use CultuurNet\UDB3\Event\CommandHandlers\UpdateOnlineUrlHandler;
+use CultuurNet\UDB3\Event\CommandHandlers\UpdateSubEventsHandler;
+use CultuurNet\UDB3\Event\CommandHandlers\UpdateThemeHandler;
+use CultuurNet\UDB3\Event\CommandHandlers\UpdateUiTPASPricesHandler;
+use CultuurNet\UDB3\Event\Productions\ProductionRepository;
+
+final class EventCommandHandlerProvider extends AbstractServiceProvider
+{
+    protected function getProvidedServiceNames(): array
+    {
+        return [
+            UpdateSubEventsHandler::class,
+            UpdateThemeHandler::class,
+            RemoveThemeHandler::class,
+            UpdateAttendanceModeHandler::class,
+            UpdateOnlineUrlHandler::class,
+            DeleteOnlineUrlHandler::class,
+            UpdateAudienceHandler::class,
+            UpdateUiTPASPricesHandler::class,
+            CopyEventHandler::class,
+        ];
+    }
+
+    public function register(): void
+    {
+        $container = $this->getContainer();
+
+        $container->addShared(
+            UpdateSubEventsHandler::class,
+            function () use ($container): UpdateSubEventsHandler {
```

Comment:
I think if you use [arrow functions](https://stitcher.io/blog/short-closures-in-php) the `use` keyword is no longer needed.